GPSTest/build.gradle

```diff
@@ -3,12 +3,12 @@ import groovy.swing.SwingBuilder
 apply plugin: 'com.android.application'
 
 android {
-    compileSdkVersion 27
-    buildToolsVersion "27.0.3"
+    compileSdkVersion 28
+    buildToolsVersion "28.0.0"
 
     defaultConfig {
         minSdkVersion 14
-        targetSdkVersion 21
+        targetSdkVersion 28
```

Comment:
We can't bump `targetSdkVersion` until https://github.com/barbeau/gpstest/issues/123 is addressed (I believe it will currently crash on first execution after install).  Can you please revert this?